site/using-weave/service-management.md

```diff
@@ -38,6 +39,17 @@ With the above in place, you are ready to connect to the 'nc' service from anywh
 
 Similar NAT rules to the above can used to expose services not just to the outside world but also other, internal, networks.
 
+###<a name="export-route"></a> Exporting Services to the wider network
+
+If you want hosts that are not running Weave Net to access services
+exposed as above, on each of those machines:
```

Comment:
No, that's not what this does. The "above" section is making *individual* services accessible to the host network *without altering the network configuration* - individual services end up with an IP/port on the host network.

By contrast, in this section we show how to make *the entire weave network* (or subnets thereof) accessible. Alas it *does* require networking changes (namely the addition of a route).

NB: a prerequisite here is that the subnet in use by weave is 'free' not just on the weave peers (as normal) but also all the hosts that require access to the weave network.

Also, position wise I reckon it's confusing for this to sit between the exporting/importing sections. I'd stick it at the very end.